docs/migrating/from-gatsby.md

```diff
@@ -0,0 +1,218 @@
+---
+description: Learn how to transition an existing Gatsby project to Next.js.
+---
+
+# Migrating from Gatsby
+
+This guide will help you understand how to transition from an existing Gatsby project to Next.js. Since both frameworks are built on top of React and share similar ideas, the transition can be broken down into a series of steps. First, let's talk about running the application.
+
+## Local Development
+
+Gatsby applications start using `gatsby develop` and run at `localhost:8000`. To create and start a production build, you run `gatsby build && gatsby serve`. Your compiled code is located at `public`.
+
+Next applications start using `next` (typically scripted as `yarn dev`) and run at `localhost:3000`. To create a start a production build, run `next build && next start`. Your compiled code is located at `.next`. You are free to change `package.json` script names as you like.
+
+The first step towards migrating to Next.js is to uninstall all related Gatsby packages and install `next`. Do not remove `react` or `react-dom` from your `package.json`.
+
+```bash
+yarn add next
+```
+
+## Serving Files
+
+The `public` folder holds static assets in Next, instead of containing the compiled output. Update your `.gitignore` file to ensure `public` is tracked with source control and delete your existing Gatsby output. You can now move files from Gatsby's `static` folder to `public`.
```

Comment:
```suggestion
The `public/` folder holds static assets in Next.js, instead of containing the compiled output. Update your `.gitignore` file to ensure `public/` is tracked in source control and delete your existing Gatsby output. You can now move files from Gatsby's `static/` folder to `public/`.
```